### ISSUE TITLE
Orientation of waterway labels

### ISSUE THREAD

**Issue by DaveF63** [plepe/OpenStreetBrowser#75]

**Comment by plepe** [comment]
You are right, there is room for improvement :-) Part of the problem is, that you can't give a distance in repetition of labels (using the [leaflet.textpath](https://github.com/makinacorpus/Leaflet.TextPath) module), so I have to add spaces after the text. But then, the label won't be rendered for short lines. So that's a compromise.

I will look into the Leaflet.TextPath module if it would be possible to add a label repetition distance (I'd thought about that anyway).

**Comment by plepe** [comment]
Ah, also there's no option (yet?) to automatically turn the text ...

**Comment by plepe** [comment]
Related issue: makinacorpus/Leaflet.TextPath#26

**Comment by plepe** [comment]
I made some progress on the Leaflet.TextPath module, as you can see here: makinacorpus/Leaflet.TextPath#72 . So it seems I'll be able to implement all your wishes :-)  

It needs some testing though and some work for the integration into OpenStreetBrowser, so this will take some time until it is available.
